src/plugins/unified_search/public/saved_query_management/saved_query_management_list.scss

```diff
@@ -2,10 +2,10 @@
   // Addition height will ensure one item is "cutoff" to indicate more below the scroll
   max-height: $euiFormMaxWidth + $euiSize;
   overflow-y: hidden;
-}
 
-.kbnSavedQueryManagement__text {
-  padding: $euiSizeM $euiSizeM ($euiSizeM / 2) $euiSizeM;
+  .euiSelectableListItem {
+    border-bottom: 1px solid darken($euiColorLightestShade, 2%);
```

Comment:
I'm not sure we need this line. If it's not supported natively from EUI I hesitate to add it. It creates a double border in the footer when there's only one page. I see we either remove it altogether or include it only when there's more than one page. What do you think @davismcphee ? 

<img width="434" alt="image" src="https://github.com/elastic/kibana/assets/4016496/507704ec-c357-4ac2-8aa8-389953cafac1">
<img width="434" alt="image" src="https://github.com/elastic/kibana/assets/4016496/1548467a-ae25-496c-87ca-152e004d504e">

<img width="434" alt="image" src="https://github.com/elastic/kibana/assets/4016496/e67f78c0-f874-41eb-9a6e-7c02bfd54e9d">

Reply:
Probably the fewer overrides the better, so I'm in favour of removing it altogether. Dropped it in this commit: https://github.com/elastic/kibana/pull/170599/commits/17c60c3c91b3532569c8a12db14127d2e35f69fb.

One other thing I overrode to try to better match the designs is removing the blue background on the focused list entry. In general I think this looks better, but now the only indicator of the focused field for keyboard users is the text underline. Do you think this is fine, or should I bring back the blue background/add another indicator of some sort?

![keyboard_nav](https://github.com/elastic/kibana/assets/25592674/e51361b5-75eb-412b-ae63-86c659a7d158)

Reply:
@davismcphee I say you bring back the blue background as the _few overrides the better_ lifestyle recommends 😉.

Reply:
Done! https://github.com/elastic/kibana/pull/170599/commits/acf4cf0f96ecd198411768e427973ef42cf6c005